statler_api/models.py

```diff
@@ -61,8 +65,8 @@ def __str__(self):
 class PlayListEntryDAO(models.Model):
     """Glues together Plays and PlayLists in an optionally ordered fashion."""
 
-    play_list = models.ForeignKey(PlayListDAO)
-    play = models.ForeignKey(PlayDAO)
+    play_list = models.ForeignKey(PlayListDAO, on_delete=models.CASCADE)
```

Comment:
This implies that the entry will delete itself if the list goes away, and NOT vice-versa. Correct?